src/ORM/DataObject.php

```diff
@@ -2706,6 +2706,12 @@ public function getCMSCompositeValidator(): CompositeValidator
 
         // Support for the old method during the deprecation period
         if ($this->hasMethod('getCMSValidator')) {
+            Deprecation::notice(
+                '5.4.0',
+                'The getCMSValidator() method is deprecated and won\'t be supported in a future major release.'
+                    . ' Override getCMSCompositeValidator() instead.',
+                Deprecation::SCOPE_GLOBAL
```

Comment:
Global scope because it's the only scope that won't include its own messaging. E.g. if we used method scope, the deprecation notice would say `DataObject::getCMSCompositeValidator() is deprecated` which is obviously not correct.